server/modules/survey/service/backup/jobs/activityLogBackupJob.js

```diff
@@ -0,0 +1,19 @@
+import Job from '@server/job/job'
+import * as ActivityLogService from '@server/modules/activityLog/service/activityLogService'
+import * as FileUtils from '@server/utils/file/fileUtils'
+
+export default class ActivityLogBackupJob extends Job {
+  constructor(params) {
+    super('ActivityLogBackupJob', params)
+  }
+
+  async execute() {
+    const { archive, surveyId, user } = this.context
+
+    const activityLogPathDir = 'activitylog'
+    const activityLogPathFile = FileUtils.join(activityLogPathDir, 'activitylog.json')
```

Comment:
wouldn't it better to have an object with backup file names?
```
BackupFileName = {
activityLog:'activitylog.json',
...
}
```

Reply:
done, good idea!